### DISCUSSION TITLE
What is wrong with all these "lora key not loaded" messages before every creation?

### DISCUSSION THREAD

**Discussion by CaBaLLaX** [runew0lf/RuinedFooocus#176] [Q&A] [answered]
Is there any way of fixing the absurd amount of messages like:
"WARNING:root:lora key not loaded: lora_unet_output_blocks_7_0_emb_layers_1.lora_up.weight
WARNING:root:lora key not loaded: lora_unet_output_blocks_7_0_in_layers_2.alpha
WARNING:root:lora key not loaded: lora_unet_output_blocks_7_0_in_layers_2.lora_down.weight
WARNING:root:lora key not loaded: lora_unet_output_blocks_7_0_in_layers_2.lora_up.weight
WARNING:root:lora key not loaded: lora_unet_output_blocks_7_0_out_layers_3.alpha..."
before every image creation when using Flux Schnell?

Why is this happening? I've seen people reporting this and fixing it on ComfyUI, but I wanted to find a solution for RuinedFooocus. Any ideas?

**Comment by runew0lf** [comment, marked as answer]
Using the wrong type of lora for the model, ie schnell lora, flux.dev, using dev lora with a gguf file. this is where the errors come in

**Reply by CaBaLLaX** [reply]
Hey, runew0lf, on a quick different note (only not to open up a new thread or something), would you know where I could read about Clip_L and all the GGUF Encoders relation between them? It seems I can only use "t5-v1_1-xxl-encoder-Q3_K_S.gguf", I've tried other versions (like Q6_K and Q3_K_M) and looks like Ruined only accepts the one version aforementioned. Why is that or what am I doing wrong?

**Reply by yownas** [reply]
There are some info on the wiki. https://github.com/runew0lf/RuinedFooocus/wiki/Flux-%E2%80%90-GGUF-and-unet-safetensors

RF should be able to handle all those. Do you get any error in the console when it fails?

**Reply by CaBaLLaX** [reply]
Yes, I get an error and couldn't understand why it was somehow "fixed" on this K_S version. Now, reading the Wiki you mentioned, I see that the "settings.json" has to be changed. This was my mistake. Thanks for letting me know! I'm gonna test it making these changes.
But LoRAs are not supported, right?

**Reply by yownas** [reply]
There is limited support for LoRAs. Models in fp8, fp16 or fp32 should work. There is no support for nf4, or gguf-models. Haven't had time to dig down exactly why gguf won't work, but my guess is that they just are stored in a different format and just can't get mixed with (most?) LoRAs as is.
